### PR TITLE
NEXUS-21938 disabled Conan plugin in HA-Cluster env

### PR DESCRIPTION
**Pull request by igorudovika**
The Conan plugin will not been verified in a HA-Cluster environment, either by manual tests or automatic tests. We should disable the Conan plugin when HA-Cluster is enabled.

### CODE REVIEW

File: nexus-repository-conan/src/test/java/org/sonatype/repository/conan/internal/proxy/matcher/ConanProxyRecipeTest.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class ConanProxyRecipeTest
+    extends TestSupport
+{
+  private static final String CONAN_FORMAT = "conan";
+
+
```
Comment (DreamVan) [inline nexus-repository-conan/src/test/java/org/sonatype/repository/conan/internal/proxy/matcher/ConanProxyRecipeTest.java]: trivial: extra line

File: nexus-repository-conan/src/test/java/org/sonatype/repository/conan/internal/proxy/matcher/ConanProxyRecipeTest.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  @Mock
+  private ConanFormat conanFormat;
+
+  @Mock
+  private HighAvailabilitySupportChecker highAvailabilitySupportChecker;
+
+  @Mock
+  private ConanProxyApiV1 conanApiV1;
+
+
```
Comment (DreamVan) [inline nexus-repository-conan/src/test/java/org/sonatype/repository/conan/internal/proxy/matcher/ConanProxyRecipeTest.java]: trivial: extra line

File: nexus-repository-conan/src/test/java/org/sonatype/repository/conan/internal/proxy/matcher/ConanProxyRecipeTest.java (hunk 1 above)
Comment (igorudovika) [reply nexus-repository-conan/src/test/java/org/sonatype/repository/conan/internal/proxy/matcher/ConanProxyRecipeTest.java]: fixed ea158a824088309928810c637aa84af43f8bf1cf

File: nexus-repository-conan/src/test/java/org/sonatype/repository/conan/internal/proxy/matcher/ConanProxyRecipeTest.java (hunk 2 above)
Comment (igorudovika) [reply nexus-repository-conan/src/test/java/org/sonatype/repository/conan/internal/proxy/matcher/ConanProxyRecipeTest.java]: fixed ea158a824088309928810c637aa84af43f8bf1cf

File: nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/proxy/v1/ConanProxyRecipe.groovy [hunk 3]
```diff
@@ -81,6 +94,7 @@ class ConanProxyRecipe
 
     builder.route(new Route.Builder()
         .matcher(BrowseUnsupportedHandler.MATCHER)
+        .handler(highAvailabilitySupportHandler)
```
Comment (sergeyonofreychuck) [inline nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/proxy/v1/ConanProxyRecipe.groovy]: Is HA handler after SecurityHandler in the chain.

Comment (igorudovika) [reply nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/proxy/v1/ConanProxyRecipe.groovy]: After additional testing moved handler into v1 Controller. Not it is explicit after Security handler 72f00903464847a5a37566837c1bfffe22804906

**Pull request merged**